README.md

```diff
@@ -35,15 +35,18 @@ See [the pip documentation] if you need to learn more about pip.
 
 ```
 cd
+become
+forks
 list
 list groups
 serial
+!  - force shell module instead of the ansible module of the same name (!yum update -y)
 ```
 
 ## Usage:
 
 ```
-[root@aaa ~]# ansible-shell
+[root@aaa ~]# ansible-shell --one-line
```

Comment:
The output format currently in the README appears new to me, but I'd like to use it if I can.
How do you achieve the output formatting shown in the README ? ::

```
UP ***********
...snip...
FAILED *******
DOWN *********
```

`--one-line` was my attempt to make it more like the above, and it works for me? ::
```
# ansible --version
ansible 1.9.3
  configured module search path = None

# ansible --help | fgrep one-line
  -o, --one-line        condense output
```

man ref: https://github.com/ansible/ansible/blob/v1.9.3-1/docs/man/man1/ansible.1.asciidoc.in#L79